### PR TITLE
add an operation to (re)?generate setup_env.sh

### PR DESCRIPTION
**Pull request by ckuethe**
implements "pybombs prefix write-env"

### CODE REVIEW

File: pybombs/commands/prefix.py [hunk 1]
```diff
@@ -60,6 +60,13 @@ def setup_subsubparser_init(parser):
         help="Use this to make the new prefix also a virtualenv. Args for virtualenv may be passed in here",
     )
 
+def setup_subsubparser_write_env(parser):
+    parser.add_argument(
+        '-p', '--prefix',
```
Comment (mbr0wn) [inline pybombs/commands/prefix.py]: `-p` is already a global switch and shouldn't be repeated here.

File: pybombs/commands/prefix.py [hunk 2]
```diff
@@ -121,6 +128,23 @@ def _run_installsdk(self):
             self.args.sdkname[0],
         )
 
+    def _run_write_env(self):
```
Comment (mbr0wn) [inline pybombs/commands/prefix.py]: This is copy & paste of code lower down. env-writing code needs factoring out.

Comment (ckuethe) [inline pybombs/commands/prefix.py]: Noted. Will fix that shortly

**Comment by mbr0wn** [comment]
@ckuethe I saw you merged this without fixing the issues. That's not how code reviews are meant to go. Please don't dump stuff into master, we don't have a 'maint' branch for PyBOMBS. It's easy for the code to become messy, and there's really no rush to get these fixes in within days, unless they're critical bugfixes.
Please fix both issues on master, or revert your commit and provide a complete patch later.

**Pull request merged**